app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
![image](https://github.com/gdumani/blogapp/assets/93039359/f8b81afe-6464-45f4-887e-3d7e9b871323)

- [ ] It would be nice if you follow the project requirements and add the following methods in your models to make your models do some automated work :+1: and to follow the project requirements :blush:

Reply:
Sorry Hammas, I sent this pull request on the wrong link, this was just to check the models, the rest of the code is in another pull request